src/components/TokenForm/TokenForm.tsx

```diff
@@ -78,7 +82,7 @@ const TokenForm: FC<TokenFormProps> = ({
         {/* inputToken select button */}
         <div className="relative z-[1] col-start-2 row-start-1 flex items-start justify-self-end pr-4 pt-4">
           <TokenSelectButton
-            canChange={!isWithdraw && isConnected}
+            canChange={!isWithdraw && isConnected && enableModalInput}
```

Comment:
Just check addresses.length here, no?

Reply:
My bad, I missed this variable. I modified it.